tests/template/srw_resize_3d_test.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+"""PyTest for :mod:`sirepo.template.srw`
+
+:copyright: Copyright (c) 2023 RadiaSoft LLC.  All Rights Reserved.
+:license: http://www.apache.org/licenses/LICENSE-2.0.html
+"""
+from pykern.pkcollections import PKDict
+from pykern.pkunit import pkeq, pkok
+
+_N_BINS = 100
+
+
+def setup_module(module):
+    import os
+
+    # Set max to something reasonable for testing
+    os.environ.update(
+        SIREPO_JOB_MAX_MESSAGE_BYTES=str(_N_BINS * _N_BINS),
+    )
+
+
+def test_srw_resize_3d(fc):
```

Comment:
You don't need `fc`. Then you can use     

```py
pkconfig.reset_state_for_testing(
        dict(
            SIREPO_JOB_MAX_MESSAGE_BYTES=str(_N_BINS * _N_BINS), 
    )
)
```

This keeps all the data local to the function.

Reply:
[d695f0d](https://github.com/radiasoft/sirepo/pull/6186/commits/d695f0d9392f805ae540aa759eba134644d93717)